src/librustdoc/html/render/print_item.rs

```diff
@@ -784,13 +786,22 @@ fn item_trait(w: &mut Buffer, cx: &mut Context<'_>, it: &clean::Item, t: &clean:
     }
 
     // Output the documentation for each function individually
-    if !required_methods.is_empty() {
+    if !required_methods.is_empty() || must_implement_one_of_functions.is_some() {
         write_small_section_header(
             w,
             "required-methods",
             "Required Methods",
             "<div class=\"methods\">",
         );
+
+        if let Some(list) = must_implement_one_of_functions.as_deref() {
+            write!(
+                w,
+                "<div class=\"stab must_implement\">At least one of `{}` methods is required.</div>",
```

Comment:
I feel like it might read better as

```suggestion
                "<div class=\"stab must_implement\">At least one of the `{}` methods is required.</div>",
```

What do you think?

Reply:
Not being english-native myself, I'll truste you for the best wording. :)

To make the comparison simpler for other readers (they both look good to me):

```
1: At least one of the `x` and `y` methods is required.
2: At least one of `x` and `y` methods is required.
```

Reply:
I've done this in ed8c2c28f0f0072dbb0ceae36b278d681d997fad